frontend/app/routes/$lang/_protected/access-to-governmental-benefits/view.tsx

```diff
@@ -88,6 +88,12 @@ export default function AccessToGovernmentalsBenefitsView() {
   const hasDentalPlans = personalInformation.federalDentalPlanId ?? personalInformation.provincialTerritorialDentalPlanId;
 
   if (hasDentalPlans) {
+    const federalSocialProgramName =
+      federalSocialProgramsList.filter((federalSocialProgram) => federalSocialProgram.id === personalInformation.provincialTerritorialDentalPlanId).map((federalSocialProgram) => getNameByLanguage(i18n.language, federalSocialProgram))[0] ?? ' ';
+    const provincialAndTerritorialProgramName =
+      provincialAndTerritorialProgramsList
+        .filter((provincialAndTerritorialProgram) => provincialAndTerritorialProgram.id === personalInformation.provincialTerritorialDentalPlanId)
+        .map((provincialAndTerritorialProgram) => getNameByLanguage(i18n.language, provincialAndTerritorialProgram))[0] ?? ' ';
```

Comment:
Unless there's a reason why we default to a space:

```suggestion
        .map((provincialAndTerritorialProgram) => getNameByLanguage(i18n.language, provincialAndTerritorialProgram))[0] ?? '';
```

Reply:
Implemented in commit ca88cab054651d5f009e974a192602f99e335f75.